### ISSUE TITLE
Extralinks - tag based filter

### ISSUE THREAD

**Issue by AntonAndreevichMoroz** [Ylianst/MeshCentral#2410]
In the documentation, I read that you can display external links for specified devices or device groups. Is it possible to filter based on tags? For example, if there is a tag HTTP8080, then show the external link.

**Comment by Ylianst** [comment]
Can you point to the documentation? I am not sure what you are referring to. Thanks.

**Comment by AntonAndreevichMoroz** [comment]
Here https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json
Part of the deviceMeshRouterLinks section.
If we add a custom protocol, we can select in the filters for which devices or groups to show this link.
Can we filter based on device tags?

**Comment by Ylianst** [comment]
Ok, so you want a MeshCentral Router link to only show up if an agent has a specific tag assigned to it?

So, it would be something like this?

```
      "deviceMeshRouterLinks": {
        "extralinks": [
          {
            "name": "HTTP",
            "protocol": "http",
            "port": 80,
            "filter": [ "tag:abc" ]                   <---------------
          }
        ]
      }
```

If that is what you want, yes, I can add that.

**Comment by AntonAndreevichMoroz** [comment]
Yes.  This is what I meant.  It will be great.  You have a very good product.

**Comment by Ylianst** [comment]
Thanks. The feature is now in GitHub, will be published with MeshCentral v0.8.1. Once published let me know if that works.

**Comment by AntonAndreevichMoroz** [comment]
Thanks. Everything works fine as intended.
